discordia/client/Multipart.lua

```diff
@@ -0,0 +1,16 @@
+local Multipart = {}
```

Comment:
I was following other lib's structure, the same way I got inspired on doing this.